### DISCUSSION TITLE
Nuget Package for TextMate support

### DISCUSSION THREAD

**Discussion by JaggerJo** [AvaloniaUI/AvaloniaEdit#182] [Q&A] [answered]
Seems like Text Mate support is not included in the Nuget Package.

**Comment by danipen** [comment]
That's true. As pointed out [here](https://github.com/AvaloniaUI/AvaloniaEdit/discussions/179), @Takoooooo is working on it.

**Comment by Takoooooo** [comment, marked as answer]
Hey, we have [published](https://github.com/AvaloniaUI/AvaloniaEdit/releases/tag/0.10.12) NuGet-s recently, feel free to use 😃
